src/project/internal/notationproject.cpp

```diff
@@ -345,7 +345,7 @@ mu::Ret NotationProject::createNew(const ProjectCreateOptions& projectOptions)
         excerpt->notation()->viewState()->makeDefault();
     }
 
-    masterScore->setSaved(true);
+    masterScore->setSaved(false);
```

Comment:
But indeed using `setSaved(true)` doesn't make much sense, that score clearly has never been saved

Reply:
I think the idea was that newly created scores were marked as "clean" since you haven't yet changed anything in them, so they don't contain any data, so no data is lost if you close/discard them. However, that is not entirely true since you may have entered information in the New Score dialog, and that information would be lost. So I wouldn't mind if we change this.

Reply:
Not just information in the wizard, also a number of instruments (unless using a template)